### DISCUSSION TITLE
[Feature] Support for Photospheres

### DISCUSSION THREAD

**Discussion by dominikheinz** [immich-app/immich#2610] [Feature Request]
### The feature

**Summary:**
Many smartphones such as the Pixel 6 are able to capture photo-spheres. 
The gallery app allows to view photo-spheres properly, where the user can look around in 3D-space.
It would be great to have support for this in Immich. Currently Immich just displays a flat stretched-out image instead.
I attached an example what I would expect instead:

Current:
![](https://i.imgur.com/sMcDewl.png)

Expected:
https://i.imgur.com/ogfnL93.gif

Examples of photo-spheres can be found here: https://commons.wikimedia.org/wiki/Category:Photo_Sphere
When it comes to detecting these kind of pictures, the EXIF data usually contains a field `Projection Type: equirectangular`, which could be used for identification. 

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by uhthomas** [comment]
Fixed by #3098.
